File: challenges/1-produto-simples/javascript/g101/produtoSimples.js
const getValues = () => {
  /*retorna um array de tipo string removendo os 2 primeiros elementos */
  let numbers = process.argv.slice(2);
  numbers[0] = parseInt(numbers[0]);
  numbers[1] = parseInt(numbers[1]);
  return numbers;
};

const calculateProduct = () => {
  const [value1, value2] = getValues(); /*destructing no array*/
  const PROD = value1 * value2;
  console.log(`produto: ${PROD}`);
};
calculateProduct();

File: challenges/1-produto-simples/java/diegooadb/ProdutoSimples.java
public class ProdutoSimples {
    public static void main(String[] args) {
        int prod = 1;
        for (String arg : args) {
            int intValue = Integer.parseInt(arg);
            prod *= intValue;
        }
        System.out.printf("produto: %d", prod);
    }
}
